.clang-tidy

```diff
@@ -6,23 +6,16 @@ Checks: '
          -clang-analyzer-security.insecureAPI.rand,
          -readability-implicit-bool-conversion,
          -cppcoreguidelines-pro-type-reinterpret-cast,
-         -readability-misleading-indentation,
          -modernize-pass-by-value,
          -cppcoreguidelines-pro-bounds-constant-array-index,
          -cppcoreguidelines-pro-type-static-cast-downcast,
-         -misc-unused-parameters,
          -cppcoreguidelines-pro-bounds-array-to-pointer-decay,
          -hicpp-no-array-decay,
-         -readability-named-parameter,
          -cert-dcl58-cpp,
-         -clang-analyzer-optin.cplusplus.VirtualCall,
          -modernize-avoid-bind,
          -hicpp-no-assembler,
```

Comment:
We don't use assembler anywhere now, right? (sorry for opening another gate to hell)

Reply:
*opening the gate*

Reply:
Narf ... clang tidy feels like it finding new instances of `misc-unused-parameters`.

Reply:
Once again, what the heck is wrong with clang-tidy?

Reply:
Assembler rule is gone.